### DISCUSSION TITLE
Original Source of Image

### DISCUSSION THREAD

**Discussion by boxofbytes** [bigratmonster/bigrat.monster#61] [General]
wheres the original image of the fat rat
this is how far i got (may 3 2020)
https://steamcommunity.com/sharedfiles/filedetails/?id=2082643526

**Comment by bigratenthusiast** [comment]
this is very interesting and i think the oldest lead we've got. good work ‼ i think a possible next step could be to ask the [person who uploaded it](https://steamcommunity.com/id/Giveup12) on steam where they found it

**Reply by boxofbytes** [reply]
ask the person then

**Comment by boxofbytes** [comment]
you can reverse search with yandex but idk how to sort by oldest so its kinda useless,
also has anyone asked the person where they got the image from

**Comment by literally1984** [comment]
Found this [reddit post](https://www.reddit.com/r/AbsoluteUnits/comments/edna9c/the_giant_rat_that_makes_all_of_the_rules/) with a picture of the rat from **December 20, 2019**.

**Reply by literally1984** [reply]
Turns out that the poster is still active on Reddit - I've shot them a PM so hopefully they respond soon

**Reply by boxofbytes** [reply]
did they respond yet?

**Reply by literally1984** [reply]
sadly, no

**Comment by boxofbytes** [comment]
good work friend

**Comment by boxofbytes** [comment]
any updates???????? has anyone asked the steam person yet pls do
